src/test/resources/scenario/FabCar.feature

```diff
@@ -29,6 +29,26 @@ Feature: Configure Fabric using SDK and submit/evaluate using a network Gateway
 		    }
 		    """
 
+	Scenario: Using a Gateway without Wallet I can submit and evaluate transactions on instantiated node chaincode
+		Given I have a gateway as user User1 using the tls connection profile without a Wallet
```

Comment:
Maybe `I have a gateway as identity User1 using the tls connection profile` to avoid the `without a Wallet` wording?

Reply:
For the scenario part, can I rephrase it to: "Using a Gateway built with X509Identity I can submit and evaluate transactions on instantiated node chaincode"?